release/Magarena/scripts/Pyromancer_s_Assault.groovy

```diff
@@ -0,0 +1,23 @@
+[
+    new OtherSpellIsCastTrigger() {
+        @Override
+        public MagicEvent executeTrigger(final MagicGame game,final MagicPermanent permanent, final MagicCardOnStack cardOnStack) {
+            return (permanent.isController(cardOnStack.getController()) && permanent.getController().getSpellsCast() == 1) ?
```

Comment:
this check can be moved to an accept method, then executeTrigger does not need to return MagicEvent.NONE

Reply:
Moved. I've notices similar deficiency in other existing groovy files. Might be worth some future refactoring...